### DISCUSSION TITLE
Allowed connections panel in statistics

### DISCUSSION THREAD

**Discussion by ghost** [tnodir/fort#307] [Ideas]
Hello, thank you for creating this app. I've been looking something to replace WFC and Netlimiter. 

Could you please make an "Allowed connections" panel inside the Statistics window, which shows the same type of information as in "Blocked connections". Basically Traffic and Blocked connections stay as is, and adding an Allowed connections panel in between them, with exactly the same columns as in Blocked connections. 

Also could you please add a Statistics button next to the Groups and Services button in the top right corner?

Looking forward to new updates. Thank you.

**Comment by tnodir** [comment]
See https://github.com/tnodir/fort/issues/27 and https://github.com/tnodir/fort/issues/295

**Comment by tnodir** [comment]
Hello and welcome.

> Also could you please add a Statistics button next to the Groups and Services button in the top right corner?

But there is a Menu button on right side of Services, where you can click on Statistics button.

**Comment by ghost** [comment]
Thanks for replying. I hope you implement allowed connections. 

I know the button is under menu, but I think having a quicker access would be beneficial since checking incoming/outgoing connections on a daily basis is probably accessed more frequently than groups and services.

**Reply by tnodir** [reply]
> I know the button is under menu, but I think having a quicker access would be beneficial

There is a Tray Event Actions in the Options: Interface. And Statistics window is configured for mouse Middle click by default .

What do you think, if same actions will be enabled for Menu button in the Programs?

So, you can open Statistics by middle click on the Menu button.

**Reply by tnodir** [reply]
> So, you can open Statistics by middle click on the Menu button.

Implemented in [v3.14.3](https://github.com/tnodir/fort/releases/tag/v3.14.3).
